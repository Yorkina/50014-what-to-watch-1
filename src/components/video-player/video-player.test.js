import React from 'react';
import renderer from 'react-test-renderer';
import VideoPlayer from './video-player.jsx';


describe(`VideoPlayer`, () => {
  it(`should render`, () => {
    const tree = renderer
      .create(<VideoPlayer
        src={`https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`}
        preview={`img/macbeth.jpg`}
        isPlaying={false}
      />).toJSON();

    expect(tree).toMatchSnapshot();

  });
});
