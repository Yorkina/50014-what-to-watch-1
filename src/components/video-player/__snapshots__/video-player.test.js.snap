// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoPlayer should render 1`] = `
<video
  height={175}
  loop={true}
  muted={true}
  poster="img/macbeth.jpg"
  src="https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4"
  width={280}
/>
`;
